src/schemas/json/project.json

```diff
@@ -6,6 +6,62 @@
 
 	"definitions": {
 		"compilationOptions": {
+			"description": "[Deprecated] Options that are passed to the compiler.",
```

Comment:
BTW it is typical for deprecated stuff to state what the replacement is. E.g.:

```
"description": "[Deprecated] Options that are passed to the compiler. Use 'buildOptions' instead.",
```

(If there's really no alternative API, you could say "There is no recommended alternative.")